src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorApiMessageBuilder.java

```diff
@@ -360,7 +360,7 @@ public static String createBatteryMeasurement_212(String batteryValue, String ti
    * @return The payload for the critical alarm creation.
    */
   public static String createCriticalAlarm_301(String type, String text, String time) {
-    return "301," + type + "," + text + "," + time;
+    return "301," + type + ",\"" + text + "\"," + time;
```

Comment:
could call one more function, this way any future changes only require modification of a single line: 
``` java
  /**
   * Create a generic alarm.
   *
   * @param alarmLevel The Cumulocity severity level  of the alarm.
   * @param type The type (fragment/series) of the alarm.
   * @param text The text (hint) of the alarm.
   * @param time The time of the alarm.
   * @return The payload for the critical alarm creation.
   */
public static String createAlarm(String alarmLevel, String type, String text, String time) {
return alarmLevel + "," + type + ",\"" + text + "\"," + time;
}
```

Reply:
Added helper method